chart/githubconnector/values.yaml

```diff
@@ -0,0 +1,26 @@
+# Default values for web-hook-chart.
```

Comment:
Please remove all the comments from this file (but remember to preserve proper spacing and clustering).